client/boot/index.js

```diff
@@ -52,6 +53,9 @@ function init() {
 
 	debug( 'Starting Calypso. Let\'s do this.' );
 
+	// prune sync-handler records
+	syncHandler.pruneRecordsFrom();
```

Comment:
I too have wanted to bring in some time constants, but I prefer basic constants instead of a library to produce them.

On a related note I had this discussion (with myself) when I wrote the interval timer. After I thought about the actual problem domain, I couldn't actually find any reason to allow arbitrary time intervals. In other words, it seemed like the most likely uses would only occupy a few different intervals, `EVERY_SECOND`, `EVERY_FIVE_SECONDS`, `EVERY_MINUTE`, `EVERY_DAY`, etc… In case we could argue that we needed this to happen every three seconds, the likely response could have been, "no we probably don't."

It might be the same here too. How many different points do we reasonably predict we will need to purge the cache out? I bet we could make the case here for no more than a few points, such as `BEFORE_YESTERDAY`, `BEFORE_LAST_HOUR` or worded differently `ONE_DAY`, `ONE_HOUR`, and maybe one or two more intervals.

Having a few reasonable constants can considerably simplify the system we're building, and it makes validating the input trivial too.